packages/neuron-ui/src/utils/const.ts

```diff
@@ -70,3 +70,4 @@ export const LIGHT_NETWORK_TYPE = 2
 export const METHOD_NOT_FOUND = -32601
 
 export const MAX_M_N_NUMBER = 255
+export const MS_PER_EPOCHS = 4 * 60 * 60 * 1000
```

Comment:
`MILLISECONDS` is more clear in this context, and `PER EPOCHS` should be `PER EPOCH`

Reply:
`4` can be replaced by `HOURS_PER_EPOCH` at https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/src/utils/const.ts#L26